nemo_tools/README.rst

```diff
@@ -0,0 +1,14 @@
+**nemo_tools**
+==========================
+
+Introduction
+------------
+
+NeMo's nemo_tools is a python package that is installed with the nemo_toolkit. See :ref:`main-readme` for installation guide.
+
+Additional requirements
+------------------------
+
+1) pynini=2.1.3
+
+setup.sh installs all additional requirements.
```

Comment:
Does it still hold if word2num is removed?

Reply:
yes